salt/modules/file.py

```diff
@@ -2449,6 +2449,8 @@ def blockreplace(path,
         content='',
         append_if_not_found=False,
         prepend_if_not_found=False,
+        insert_before_match=None,
```

Comment:
These need to go at the end of the list so we don't cause trouble with people calling these kwargs positionally.